suite-native/biometrics/src/components/BiometricOverlay.tsx

```diff
@@ -34,6 +39,13 @@ export const BiometricOverlay = ({
 }: BiometricOverlayProps) => {
     const { applyStyle } = useNativeStyles();
 
+    // Ignore device connection in biometrics overlay (note for code review: is this actually necessary?)
+    useEffect(() => {
+        stopDeviceConnectionListening();
+
+        return () => startDeviceConnectionListening();
+    }, []);
```

Comment:
feel free to ignore the comment, just thinking loud :D 

but pattern 

```
  useEffect(() => {
        stopDeviceConnectionListening();

        return () => startDeviceConnectionListening();
    }, []);
```

seems to be pretty handy, and have feeling that it might be needed at more places in the future, maybe it could be a hook, smth like `useFocus(Something :D )`

Reply:
actually I see you are also using it in [useDisableDeviceConnectionOnRouteBlacklist](https://github.com/trezor/trezor-suite/pull/20081/files#diff-76c790c91a22bb3d0a60f3c8bedf447fe949ed4c6380ee5103020d509f3a7782R37)

```

    useFocusEffect(
        useCallback(() => {
            if (shouldDisableConnection) {
                stopDeviceConnectionListening();

                return () => {
                    startDeviceConnectionListening();
                };
            }
        }, [shouldDisableConnection]),
```

Reply:
I'm afraid this pattern is not working very well - imagine the same effect in multiple components. If one of those components unmount, it starts device connection listening even though other components are still mounted.

Reply:
Redundant, stopping listening is not used anymore. See https://github.com/trezor/trezor-suite/pull/20081/commits/0054b20586c7a0ef85bdd6b5f788ed003083a561